conda/recipes/librmm/meta.yaml

```diff
@@ -1,6 +1,6 @@
 # Copyright (c) 2018-2023, NVIDIA CORPORATION.
 
-{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') %}
+{% set version = environ.get('RAPIDS_PACKAGE_VERSION', '0.0.0.dev').strip('"').lstrip('v') %}
```

Comment:
Can we remove the default value, thereby making this environment variable a hard requirement? We've done this already for `RAPIDS_CUDA_VERSION` and `RAPIDS_DATE_STRING`. The only reason I didn't remove this default at the same time was that I didn't understand if versioneer or some other piece of our process needed it to have a default value. We've ripped that out so I see no reason for a default now.

Reply:
I'd be fine getting rid of the default.

Reply:
@ajschmidt8 any objections?

Reply:
no objections from me.